src/components/DocExplorer.js

```diff
@@ -56,7 +56,10 @@ export class DocExplorer extends React.Component {
   constructor() {
     super();
 
-    this.state = { navStack: [] };
+    this.state = {
+      navStack: [],
+      searchValue: null
```

Comment:
I think searchValue should probably be part of the nav stack rather than it's own thing, so that you can search, click on something, click back and be where you were before.

This could be done in a separate PR though.